modules/localgov_roles/src/RolesHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\localgov_roles;
+
+use function call_user_func;
```

Comment:
Do we really need ```use function call_user_func```?

Reply:
Probably not. intelephense added it, like it does other use statements, but not usually for non-namespaced global functions, and I didn't remove it :shrug: I only used call_user_func rather than a variable function name out of readability.

Out of curiosity I had a quick look round some vendor puts a use statement in, most doesn't, drupal core like the rest of vendor sometimes prepends the function with a \ for namespace, sometimes doesn't.